src/services/codefixes/convertToTypeOnlyImport.ts

```diff
@@ -104,7 +105,8 @@ function canConvertImportDeclarationForSpecifier(specifier: ImportSpecifier, sou
     const checker = program.getTypeChecker();
     for (const specifier of nonTypeOnlySpecifiers) {
         const isUsedAsValue = FindAllReferences.Core.eachSymbolReferenceInFile(specifier.name, checker, sourceFile, usage => {
-            return !isValidTypeOnlyAliasUseSite(usage);
+            const symbol = checker.getSymbolAtLocation(usage);
+            return !isValidTypeOnlyAliasUseSite(usage) || !!symbol && !checker.getTypeOnlyAliasDeclaration(symbol, SymbolFlags.Value);

```

Comment:
This needs a few more tests. I think you may still need to `skipAlias` and check flags like you had in the previous commit as part of this condition. It’s too complicated to guess at without test coverage though. Here are some cases, assuming the same `b.ts` and test execution.

Should use `import type` (`f` has no reified export):
```ts
// @filename: /a.ts
function f() {}
export interface I {}
export type { f };
```

```ts
// @filename: /a.ts
export interface f {}
export interface I {}
```

Should use `import { type` (`f` has reified export):
```ts
// @filename: /z.ts
export function f() {}

// @filename: /a.ts
import { f } from "./z";
export interface I {}
type f = number;
export { f };
```

Reply:
Should we have a `QF' suggestion in the last example?

Reply:
I was assuming there would still be a `b.ts` paired with the last example that imports and re-exports `f` and `I`